INSTALL.md

```diff
@@ -99,7 +99,7 @@ your project:
 package: ( your project's import path ) # e.g. github.com/foo/bar
 import:
 - package: k8s.io/client-go
-  version: v2.0.0
```

Comment:
We clearly need to "fix" v2 but I'm a little worried that we are prescribing client v6. It seems live the "right" version is the fully compatible version for your kubernetes cluster or the latest such version if you have multiple kubernetes clusters with different versions. I am not seeing anything in the docs which explains that or that this the line you would want to change.

Reply:
It's documented in README.md (https://github.com/kubernetes/client-go/pull/347/files#diff-04c6e90faac2675aa89e2176d2eec7d8R82). But your comment shows once again that all this is far too complicated and spread over too many files. We cannot change this here in the PR, but I am fore a major rewrite of the docs here. Compare https://github.com/kubernetes/client-go/pull/337 from your colleague who struggled as well from this documentation.